outlines/structure.md

```diff
@@ -0,0 +1,56 @@
+# Application structure and code style
+
+1. What's different between Meteor and other kinds of apps?
+    1. All JavaScript, enables code sharing
+    2. Structure is flexible
+    3. Currently everything is about magic file names
+    4. App scope vs. package scope vs. file scope variables
+2. Client vs. server vs. both
+3. Smooth path from prototype to production, three sizes of apps, S, M, L
+4. General stuff
+    1. Filenames - named the same as the export
+        1. Every file exports one thing
+    2. Avoid depending on load order via namespacing and Meteor.startup
+        1. If you depend on load order, your best bet is a package
+    3. Naming guidelines for everything
+        1. https://github.com/meteor/guide/issues/9#issuecomment-148800085
```

Comment:
I think our idea for naming methods was that we'd be referencing them via the JS namespace anyway, and the string name would be something used internally for REST+DDP paths.

So in this case, the method would be `Posts.methods.list = new Method({name: 'posts/methods/list`})` or something (have we decided on a sub-namespace for methods @stubailo?).

I'm definitely on board with the three names you mentioned being the same. I'm not quite sure I see that the method is quite as tightly bound/coupled as the route/template/publication though. As an example, I'd probably want to call a method something like `Posts.insert` but the template/route would probably be called `Posts.new` or something.